tests/allowed-deprecations.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "location": "Zenstruck\\Foundry\\Persistence\\PersistentObjectFactory",
+        "message": "The \"Zenstruck\\Foundry\\BaseFactory::new()\" method is considered final. It may change without further notice as of its next major version. You should not extend it from \"Zenstruck\\Foundry\\Persistence\\PersistentObjectFactory\".",
```

Comment:
this deprecation occurs in all tests, that's why I've added to the "allowed" list